setup.py

```diff
@@ -27,7 +28,8 @@
             'em = materials_io.electron_microscopy:ElectronMicroscopyParser',
             'image = materials_io.image:ImageParser',
             'crystal = materials_io.crystal_structure:CrystalStructureParser',
-            'dft = materials_io.dft:DFTParser'
+            'dft = materials_io.dft:DFTParser',
+            'ase = materials_io.ase:AseParser'
```

Comment:
Did my docs say anything about adding the parser to the readthedocs? There is a page in the docs directory about describing the available parsers.